pilot/pkg/features/pilot.go

```diff
@@ -295,6 +295,18 @@ var (
 	JwtPolicy = env.RegisterStringVar("JWT_POLICY", jwt.JWTPolicyThirdPartyJWT,
 		"The JWT validation policy.")
 
+	// Default timeout for virtual services if a timeout is not configured in virtual service. It defaults to zero
```

Comment:
Which timeout, please add it clearly, like this is the http request timeout?

Reply:
Will do